src/instance/commands/triggers/calculate.ts

```diff
@@ -28,6 +28,9 @@ export default class Calculate extends ChatCommandHandler {
       .replaceAll(',', '') // removes commas from numbers
 
     const result = evalExpression(expression)
```

Comment:
```suggestion
    const result = evalExpression(expression)
    // The following if-statement is purely an easter egg
    // It can be removed without causing any adverse affects on the bridge
```
Added a comment explaining the easter egg to remove confusion for other collaborators/users